server.js

```diff
@@ -33,7 +33,7 @@ var _ = require('underscore'),
     monthlyRegistrations = require('./controllers/monthly-registrations'),
     monthlyDeliveries = require('./controllers/monthly-deliveries'),
     exportData = require('./controllers/export-data'),
-    messages = require('./controllers/messages'),
+    messages = require('./controllers/message-utils'),
```

Comment:
IMO naming the file `controllers/messages.js` is better.

It seems reasonable to me that any controller could be accessed internally, as this should be roughly equivalent to accessing it via it's corresponding endpoint.  Of course, accessing a controller via `require()` will bypass auth, which may or may not be desirable in general.